vector-im/element-android
# Voice Broadcast - Improve live indicator icon rendering

## Type of change


## Content

Improve the rendering of the "live" indicator icon in the timeline.

## Motivation and context

Continue #7127 

The live indicator icon is not updated according to the following rules:

<img width="703" alt="image" src="https://user-images.githubusercontent.com/11990514/201625419-0d30b74a-6704-4c7e-8e6a-2799680338b4.png">

## Screenshots / GIFs



## Tests

- Start recording a Voice Broadcast from a device A
- Open the room timeline on a device B and observe that the indicator is visible & grey
- Start listening the VB from the device B
- Observe that the live indicator is visible & red while buffering and when playing a new chunk
- Move backward in the playback & observe that the indicator switches to grey
- Move forward and observe that the indicator is red only if you're in the last chunk (last 2 minutes)
- Pause and observe that the indicator is grey, resume and observe that it is red only if you're in the last chunk (last 2 minutes)
- From the device A, pause the VB and observe on device B that the icon is grey and stays grey regardless of the playing state
- Resume the recording and observe that the indicator is red only if you're in the last chunk (last 2 minutes)
- Stop the recording, the indicator should disappear for both devices

## Tested devices

- OS version(s):

## Checklist
